frontend/src/queries/nodes/InsightQuery/utils/queryNodeToFilter.ts

```diff
@@ -127,10 +134,22 @@ export const queryNodeToFilter = (query: InsightQueryNode): Partial<FilterType>
 
     if (isTrendsQuery(query)) {
         ;(filters as TrendsFilterType).display = query.trendsFilter?.display
```

Comment:
fly-by since this bothered me when I change it last as well... we should have a `hasTrendsFilter` (or similar named) typeguard so we don't need the leading semicolon and typecasts within the conditional...

not blocking here just thinking aloud

Reply:
Oh yeah, that would definitely be better and I'll add something next time I touch `queryNodeToFilter`. If we are successful in the migration to data exploration, this function shouldn't exist for long any more however.